src/client/rewards/Rewards.js

```diff
@@ -536,16 +538,29 @@ class Rewards extends React.Component {
   };
 
   handleLoadMore = () => {
-    const { propositions, hasMore, sort, area, activeFilters } = this.state;
+    const {
+      propositions,
+      hasMore,
+      sort,
+      area,
+      activeFilters,
+      radius,
+      isSearchAreaFilter,
+    } = this.state;
     const { username, match } = this.props;
     if (hasMore) {
       this.setState(
         {
           loading: true,
         },
         () => {
-          const reqData = preparePropositionReqData({ username, match, sort, area });
+          const reqData = preparePropositionReqData({ username, match, sort, area, activeFilters });
+          const guideNames = get(activeFilters, 'guideNames');
+          const types = get(activeFilters, 'types');
+          reqData.guideNames = guideNames;
+          reqData.types = types;
```

Comment:
configure `reqData` in `preparePropositionReqData()`
see https://github.com/Waiviogit/waivio/commit/bebd1bd4dfadd78721016235f62732f8ed2dd124